src/plugins/intel_cpu/src/nodes/kernels/gather_uni_kernel.hpp

```diff
@@ -24,6 +24,7 @@
 
```

Comment:
Please update or remove the table at line 11

Reply:
Done